Misc/NEWS.d/next/Library/2024-07-21-18-03-30.gh-issue-122088.vi2bP-.rst

```diff
@@ -0,0 +1,4 @@
+:func:`@warnings.deprecated <warnings.deprecated>` now copies the
+:data:`coroutine flag <inspect.CO_COROUTINE>` flag of functions and
```

Comment:
This is inaccurate, you should just write that it marks the wrapper as coroutine too. This actual implmentation doesn't alters code flags in `inspect.markcoroutinefunction`.